src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -376,6 +376,11 @@ def test_user_standard_waveform(session):
     session.clear_user_standard_waveform()
 
 
+def test_routing_attribute(session):
```

Comment:
@marcoskirsch Based on what you said, I expected this to fail?

Reply:
I'm not sure. NI-FGEN devices had special circuitry for clock signals. This may be one of those.
Can you try with one of the triggers? Maybe [`nifgen.Session.digital_edge_start_trigger_source`](http://nimi-python.readthedocs.io/en/master/nifgen/attributes.html#nifgen.Session.digital_edge_start_trigger_source)